### PR TITLE
New error ratio metrics (gauges) for 4xx and 5xx service responses

### PR DESCRIPTION
**Pull request by viktorjonsson**

### CODE REVIEW

File: modules/metrics/src/main/java/com/spotify/apollo/metrics/semantic/SemanticRequestMetrics.java [hunk 1]
```diff
@@ -92,9 +101,15 @@ public void response(Response<ByteString> response) {
     timerContext.ifPresent(Timer.Context::stop);
 
     StatusType.Family family = response.status().family();
-    if (family == SERVER_ERROR) {
+    if (family != INFORMATIONAL && family != SUCCESSFUL) {
```
Comment (gilles) [inline modules/metrics/src/main/java/com/spotify/apollo/metrics/semantic/SemanticRequestMetrics.java]: redirects should not count as errors either

**Comment by gilles** [comment]
someone beats me in making the 4xx and 5xx ratios :) thanks!

File: modules/metrics/src/main/java/com/spotify/apollo/metrics/semantic/SemanticRequestMetrics.java (hunk 1 above)
Comment (viktorjonsson) [reply modules/metrics/src/main/java/com/spotify/apollo/metrics/semantic/SemanticRequestMetrics.java]: The reason why I added two new metrics was that I didn't want to touch the error-ratio one (as was discussed in this PR https://github.com/spotify/apollo/pull/205). I'd prefer not to touch that metric.

**Comment by mattnworb** [comment]
If you want to merge this into the 1.x branch in #233, shouldn’t you merge it into master as well for whatever comes after 1.x?

**Pull request closed without merging**